connectors/bitcoin.go

```diff
@@ -174,10 +174,10 @@ func (btc *BTC) GetDerivedBitcoinAddress(userBtcRefundAddr []byte, lbcAddress []
 	return addressScriptHash.EncodeAddress(), nil
 }
 
-func DecodeBTCAddress(address string) ([]byte, error) {
+func GetBytesFromBtcAddress(address string) ([]byte, error) {
```

Comment:
I think that Decode is a better name consistent with other packages like hex.

Reply:
Renamed.